galsim/sum.py

```diff
@@ -96,11 +96,15 @@ class Sum(GSObject):
     @param gsparams         An optional GSParams argument.  See the docstring for GSParams for
                             details. [default: None]
 
-    Note: if `gsparams` is unspecified (or None), then the Sum instance inherits the same GSParams
-    as the first item in the list.  Also, note that parameters related to the Fourier-space
-    calculations must be set when initializing the individual GSObject instances that go into the
-    Sum, NOT when creating the Sum (at which point the accuracy and threshold parameters will simply
-    be ignored).
+    Note: if `gsparams` is unspecified (or None), then the Sum instance will use the most
+    restrictive combination of parameters from each of the component objects. Normally, this means
+    the smallest numerical value (e.g. folding_threshold, xvalue_accuracy, etc.), but for a few
+    parameters, the largest numerical value is used.  See GSParams.combine for details.
+
+    Furthermore, the gsparams used for the Sum (either given explicitly or derived from the
+    components) will be applied to each of the components.  It doesn't make much sense to apply
+    stricter-than-normal accuracy or threshold values to one component but not another in a Sum,
+    so this ensures that they all have consistent rendering behavior.
```

Comment:
That's actually a very good point.  While I think it most cases, it will be fine, I think we probably ought to add an option to keep the component gsparams the same in case a user is in this kind of scenario.  Maybe `propagate_gsparams=False`.  (Default `True`)

Reply:
Sounds good to me.

Reply:
Done.  478e4f2a36fea8cdf2992ed3cf89b7d9fda58805